docs/topics/modules/interpreting_stack_report.adoc

```diff
@@ -0,0 +1,47 @@
+[#interpreting_stack_report]
+= Interpreting your Stack Report
+
+The Stack Report comprises three components:
+
+== Stack report: An overall stack analysis
+The Stack report scans the total number of components in your stack, and reports those analyzed by {osio} and those unknown to it.
```

Comment:
So, the feature is called "Stack [R|r]eport", and it scans and reports? And when it reports, it generates a report?

It's funny at best and confusing at all other times. If this description is correct, a bug should be filed against the name of the feature.